<hr>

# Module 30.4: Earnings Per Share

<hr>


### LOS 30.d: Describe how earnings per share is calculated and calculate and interpret a company's basic and diluted earnings per share for companies with simple and complex capital structures including those with antidilutive securities.

**Earnings per share (EPS)** is one of the most commonly used corporate profitability performance measures for publicly traded firms (nonpublic companies are not required to report EPS data). EPS is reported only for shares of common stock (also known as ordinary stock).

A company may have either a simple or complex capital structure:

* A **simple capital structure** is one that contains no potentially dilutive securities. A simple capital structure contains only common stock, nonconvertible debt, and nonconvertible preferred stock.
* A **complex capital structure** contains *potentially dilutive securities* such as employee stock options, warrants, or convertible securities.

All firms with complex capital structures must report both basic and *diluted* EPS. Firms with simple capital structures report only basic EPS.



#### Basic EPS

The basic EPS calculation does not consider the effects of any dilutive securities in the computation of EPS:
\
\
$$
\text{basic EPS} = \frac{\text{net income - preferred dividends}}{\text{weighted average number of common shares outstanding}}
$$
    <p> <br> </p>

The current year's preferred dividends are subtracted from net income because EPS refers to the per-share earnings *available to common shareholders*. Net income minus preferred dividends is the income available to common stockholders. Common stock dividends are not subtracted from net income because they are a part of the net income available to common shareholders.

The **weighted average number** of common shares is the number of shares outstanding during the year, weighted by the portion of the year they were outstanding.

<hr>

#### Effect of Stock Dividends and Stock Splits

A s**tock dividend** is the distribution of additional shares to each shareholder in an amount proportional to their current number of shares. If a 10% stock dividend is paid, the holder of 100 shares of stock would receive 10 additional shares.

A **stock split** refers to the division of each "old" share into a specific number of "new" (post-split) shares. The holder of 100 shares will have 200 shares after a 2-for-1 split, or 150 shares after a 3-for-2 split.

The important thing to remember is that each shareholder's proportional ownership in the company is unchanged by either of these events. Each shareholder has more shares but the same percentage of the total shares outstanding. For calculating EPS, we apply stock dividends and splits retroactively to the beginning of the year, to all shares before the date of the corporate event. Prior years' weighted average number of common shares is also adjusted as if the stock split or stock dividend had occurred in the prior period, to prevent EPS from appearing to decline when these actions are largely cosmetic.

>**PROFESSOR'S NOTE**
\
For our purposes here, a stock dividend and a stock split are two ways of doing the same thing. For example, a 50% stock dividend and a 3-for-2 stock split both result in three "new" shares for every two "old" shares.    
<p> <br> </p>

The following are key things to know about calculating weighted average shares outstanding:

* The weighting system is days outstanding divided by the number of days in a year, but on the exam, the monthly approximation method will probably be used.
* Shares issued enter into the computation from the date of issuance.
* Reacquired shares are excluded from the computation from the date of reacquisition.
* Shares sold or issued in a purchase of assets are included from the date of issuance.
* A stock split or stock dividend is applied to all shares outstanding before the split or dividend and to the beginning-of-period weighted average shares. A stock split or stock dividend adjustment is not applied to any shares issued or repurchased after the split or dividend date.




<hr>

#### Example: Weighted average shares outstanding

Johnson Company has 10,000 shares outstanding at the beginning of the year. On April 1, Johnson issues 4,000 new shares. On July 1, Johnson distributes a 10% stock dividend. On September 1, Johnson repurchases 3,000 shares. Calculate Johnson's weighted average number of shares outstanding for the year, for its reporting of basic earnings per share.

**Answer:**

Shares outstanding are weighted by the portion of the year that the shares were outstanding. Any shares that were outstanding before the 10% stock dividend must be adjusted for it. Transactions that occur after the stock dividend do not need to be adjusted.

**PROFESSOR'S NOTE**
\
Think of the shares before the stock dividend as "old" shares and shares after the stock dividend as "new" shares that each represent ownership of a smaller portion of the company (in this example, 10/11ths of that of an old [pre-stock dividend] share). The weighted average number of shares for the year will be in new shares.

<table>
<tbody>
  <tr>
    <td>Shares outstanding on January 1: 10,000 × 1.10 × 12/12 of the year</td>
    <td>= </td>
    <td>11,000</td>
  </tr>
  <tr>
    <td>Shares issued April 1: 4,000 × 1.10 × 9/12 of the year</td>
    <td>= =</td>
    <td>3,300</td>
  </tr>
  <tr>
    <td>Shares repurchased September 1: –3,000 × 4/12 of the year</td>
    <td>= </td>
    <td>–1,000</td>
  </tr>
  <tr>
    <td>Weighted average shares outstanding </td>
    <td>= </td>
    <td>13,300</td>
  </tr>
</tbody>
</table>

<hr>

#### Example: Basic earnings per share

Johnson Company has net income of <FONT COLOR="#ff0000">€10,000</font>, paid €1,000 cash dividends to its preferred shareholders, and paid €1,750 cash dividends to its common shareholders. Calculate Johnson's basic EPS using the weighted average number of shares from the previous example.

**Answer:**

$\text{basic EPS} = \Large\frac{\text{net income - preferred dividends}}{\text{weighted average shares of common}}$
<p> <br> </p>

$\qquad = \Large\frac{€10,000 − €1,000}{13,300} \small = €0.68$

<p> <br> </p>

>**PROFESSOR'S NOTE**
\
Remember, the payment of a cash dividend on common shares is not considered in the calculation of EPS.

In [2]:
# Example: EPS Johnson company 

EPS_Johnson = (10000-1000)/13300
print("EPS for Johnson Company is =",round(EPS_Johnson,2), "€")

EPS for Johnson Company is = 0.68 €


#### Diluted EPS

Before calculating diluted EPS, it is necessary to understand the following terms:

* **Dilutive securities** are stock options, warrants, convertible debt, or convertible preferred stock that would *decrease* EPS if exercised or converted to common stock.
* **Antidilutive securities** are stock options, warrants, convertible debt, or convertible preferred stock that would *increase EPS* if exercised or converted to common stock.

We apply the <FONT COLOR="#ff0000"> "if converted" method</font> to examine the impact of potentially dilutive securities. This looks at what EPS would have been if the securities were converted at the start of the accounting period, regardless of whether they can actually be converted during the period.

The numerator of the basic EPS equation contains income available to common shareholders (net income less preferred dividends). In the case of diluted EPS, if there are dilutive securities, the numerator must be adjusted as follows:

* If convertible preferred stock is dilutive (meaning EPS will decrease if it is converted to common stock), the convertible preferred dividends must be added to earnings available to common shareholders.
* If convertible bonds are dilutive, then the bonds' after-tax interest expense is not considered an interest expense for diluted EPS. Hence, interest expense multiplied by $(1 – \text{the tax rate})$ must be added back to the numerator.

**PROFESSOR'S NOTE**
\
Interest paid on bonds is typically tax deductible for the firm. If convertible bonds are converted to stock, the firm saves the interest cost but loses the tax deduction. Thus, only the after-tax interest savings are added back to income available to common shareholders.

<p> <br> </p>

The basic EPS denominator is the weighted average number of shares. When the firm has dilutive securities outstanding, the denominator is adjusted for the equivalent number of common shares that would be created by the conversion of all dilutive securities outstanding (convertible bonds, convertible preferred shares, warrants, and options), with each one considered separately to determine if it is dilutive.

If a dilutive security was issued during the year, the increase in the weighted average number of shares for diluted EPS is based on only the portion of the year the dilutive security was outstanding.

Dilutive stock options or warrants increase the number of common shares outstanding in the denominator for diluted EPS. There is no adjustment to the numerator.

the **diluted EPS equation** is:

$\qquad \text{diluted EPS}  = \Large\frac{\text{adjusted income available for common shares}}{\text{weighted-average common and potential common shares outstanding}}$
\
\
where:

adjusted income available for: 
\
$\text{common shares} = \text{net income – preferred dividends}$ + dividends on convertible preferred stock + after-tax interest on convertible debt
\
\
Therefore, <FONT COLOR="#ff0000">diluted EPS</font> is:

$$
\text{adjusted income available for common shares} = \\ 
\text{net income - preferred dividends} \\
+ \text{dividends on convertible preferred stock}\\
+ \text{after-tax interest on convertible debt}
$$
\
\
The effect of conversion to common shares is included in the calculation of diluted EPS for a given security only if it is, in fact, dilutive. If a firm has more than one potentially dilutive security outstanding, each potentially dilutive security must be examined separately to determine if it is actually dilutive (i.e., would reduce EPS if converted to common stock).

<hr>

#### Example: EPS with convertible preferred stock

During 20X6, ZZZ Corp. reported net income of €4.35 million and had 2 million shares of common stock outstanding for the entire year. ZZZ's 7\%, €5 million par value preferred stock is convertible into common stock at a conversion rate of 1.1 shares for every €10 of par value. Calculate basic and diluted EPS.

In [13]:
#### Example: EPS with convertible preferred stock

# Step 1 Calculate 20X6 basic EPS
basic_EPS = (4350000-((0.07)*(5000000)))/2000000
print("The Basic EPS for the ZZZ stock is = €",basic_EPS)

The Basic EPS for the ZZZ stock is = € 2.0


**Answer:**

Step 1: Calculate 20X6 basic EPS:

\
$\qquad\text{basic EPS} = \Large\frac{€4,350,000 - (0.07)(€5,000,000)}{2,000,000} \small = €2.00$
<p> <br> </p>

Step 2: Calculate diluted EPS:

* Compute the increase in common stock outstanding if the preferred stock is converted to common stock at the beginning of 20X6: (€5,000,000 / €10) × 1.1 = 550,000 shares.
* If the convertible preferred shares were converted to common stock, there would be no preferred dividends paid. Therefore, you should add back the convertible preferred dividends that had previously been subtracted from net income in the numerator.
* Compute diluted EPS as if the convertible preferred stock were converted into common stock:
<p> <br> </p>

$\text{diluted EPS} = \Large\frac{\text{net. inc.} - \text{pref. div.} + \text{convert. pref. dividends}} {\text{wt. avg. shares} + \text{convert. pref. common shares}}$

<p> <br> </p>

$\text{diluted EPS} = \Large\frac{€4,350,000}{2,000,000 + 550,000} \small = €1.71$

<p> <br> </p>

* Check to see if diluted EPS is less than basic EPS $(€1.71 < €2.00)$. If the answer is yes, the preferred stock is dilutive and must be included in diluted EPS as computed previously. If the answer is no, the preferred stock is antidilutive, and conversion effects are not included in diluted EPS.

In [9]:
print("Basic EPS for 20X6   = €",basic_EPS)

# Step 2: Calculate diluted EPS:
diluted_EPS = 4350000/(2000000+550000)
print("Diluted EPS for 20X6 = €",round(diluted_EPS,2))

Basic EPS for 20X6   = € 2.0
Diluted EPS for 20X6 = € 1.71


A quick way to check whether convertible preferred stock is <u>dilutive</u> is to divide the preferred dividend by the number of shares that will be created if the preferred stock is converted. For ZZZ:
<p> <br> </p>
$\qquad\Large\frac{€5,000,000 x 0.07}{550,000} \small = €0.64$


In [21]:
# Print prior calculations
print("Diluted EPS for 20X6 = €",round(diluted_EPS,2))
print("Basic EPS for 20X6   = €",basic_EPS)


# Step 3: check whether convertible preferred stock is dilutive 
check_dilute = (5000000 * 0.07)/(550000)
print("\n")
print("Check if convertible preferred is dilutive = €",round(check_dilute,2))
print("Because this is less than basic EPS, the convertible preferred is dilutive.")

Diluted EPS for 20X6 = € 1.71
Basic EPS for 20X6   = € 2.0


Check if convertible preferred is dilutive = € 0.64
Because this is less than basic EPS, the convertible preferred is dilutive.


Because the number above is less than basic EPS, the convertible preferred is dilutive.

The convertible preferred is dilutive, because $(0.64 < 2.0)$
<hr>

#### Example: EPS with convertible debt

During 20X6, YYY Corp. had earnings available to common shareholders of €2.5 million and had 1 million shares of common stock outstanding for the entire year, for basic EPS of €2.50. During 20X5, YYY issued 2,000, €1,000 par, 5\% bonds for €2 million (issued at par). Each of these bonds is convertible to 120 shares of common stock. The tax rate is 30\%. Calculate the 20X6 diluted EPS.

In [78]:
#### Example: EPS with convertible debt

# Step 1:  Compute the increase in common stock outstanding 
# if the convertible debt is converted to common stock at the beginning of 20X6:


# Calculate shares issuable for debt conversion
shares_convert = 2000 * 120
print("shares issuable for debt conversion = ",'{:,.0f}'.format(shares_convert),"shares")

# Calculate increase in income
incr_income = (2000*1000*0.05) * (1 - 0.3)
print("\n")
print("increase YYY's after-tax net income for the after-tax effect of \n the decrease in interest expense")
print("")
print("interest in income = ",'{:,.2f}'.format(incr_income),"euros")

# Calculate the Diluted EPS
diluted_EPS = (2500000+70000)/(1000000+240000)
print("\n")
print("The Diluted EPS is = ",'{:,.2f}'.format(diluted_EPS),"euros")

# Calculate the per-share-impact of convertible debt
per_share_impact = 70000/240000
print("\n")
print("The per-share-impact of convertible debt is = ",'{:,.2f}'.format(per_share_impact),"euros")

shares issuable for debt conversion =  240,000 shares


increase YYY's after-tax net income for the after-tax effect of 
 the decrease in interest expense

interest in income =  70,000.00 euros


The Diluted EPS is =  2.07 euros


The per-share-impact of convertible debt is =  0.29 euros


**Answer**

Compute the increase in common stock outstanding if the convertible debt is converted to common stock at the beginning of 20X6:


$\qquad\text{shares issuable for debt conversion} = (2,000)(120) = 240,000 \text{ shares}$

If the convertible debt is considered converted to common stock at the beginning of 20X6, then there would be no interest expense related to the convertible debt. Therefore, it is necessary to increase YYY's after-tax net income for the after-tax effect of the decrease in interest expense:

$\qquad\text{increase in income} = [(2,000)(€1,000)(0.05)] (1 − 0.30) = €70,000$

Compute diluted EPS as if the convertible debt were common stock:

<p> <br> </p>

$\text{diluted EPS} = \Large\frac{\text{net. inc.} - \text{pref. div.} + \text{convert. pref. dividends}} {\text{wt. avg. shares} + \text{convert. pref. common shares}}$

<p> <br> </p>

$\text{diluted EPS} = \Large\frac{€2,500,000 + 70,000}{1,000,000 + 240,000} \small = €1.71$

<p> <br> </p>

Check to make sure that diluted EPS is less than basic EPS $(€2.07 < €2.50)$. If diluted EPS is more than the basic EPS, the convertible bonds are antidilutive and should not be treated as common stock in computing diluted EPS.

A quick way to determine whether the convertible debt is dilutive is to calculate its per share impact:

$\qquad\Large\frac{\text{convertible debt interest (1 - t)}}{\text{convertible debt shares}}$

<p> <br> </p>
If this per share amount is greater than basic EPS, the convertible debt is antidilutive, and the effects of conversion should not be included when calculating diluted EPS.

If this per share amount is less than basic EPS, the convertible debt is dilutive, and the effects of conversion should be included in the calculation of diluted EPS.

For YYY:

$\Large\qquad\frac{€70,000}{240,000} \small= €0.29$

The company's basic EPS is €2.50, so the convertible debt is dilutive, and the effects of conversion should be included in the calculation of diluted EPS.



<hr>

Stock options and warrants are dilutive only when their exercise prices are less than the average market price of the stock over the year. If the options or warrants are dilutive, use the **treasury stock method** to calculate the number of shares used in the denominator:

* The treasury stock method assumes that the funds received by the company from the exercise of the options would be used to hypothetically purchase shares of the company's common stock in the market at the average market price.
* The net increase in the number of shares outstanding (the adjustment to the denominator) is the number of shares created by exercising the options less the number of shares hypothetically repurchased with the proceeds of exercise.

<hr>

#### Example: EPS with stock options

During 20X6, XXX Corp. reported earnings available to common shareholders of €1.2 million and had 500,000 shares of common stock outstanding for the entire year, for basic EPS of €2.40. XXX has 100,000 stock options (or warrants) outstanding the entire year. Each option allows its holder to purchase one share of common stock at €15 per share. The average market price of XXX's common stock during 20X6 is €20 per share. Calculate diluted EPS.

In [75]:
# Number of common shares created if the options are exercised
Shares_if_exercised = 100000
print("Number of common shares created if the options are exercised = ",'{:,.0f}'.format(Shares_if_exercised),"shares")
print("")

# Cash inflow if the options are exercised 
cash_inflow = 15 * 100000
print("Cash inflow if the options are exercised = ",'{:,.2f}'.format(Shares_if_exercised),"euros")
print("")


# Number of shares that can be purchased with these funds 
num_shares_purchase = 1500000/20
print("Number of shares that can be purchased with these funds = ",'{:,.0f}'.format(num_shares_purchase),"shares")
print("")

# Net increase in common shares outstanding from the exercise of the stock options 
net_incr = 100000 - 75000
print("Net increase in shares outstanding from exercising stock options = ",'{:,.0f}'.format(net_incr),"shares")
print("")

# Diluted Earnings per share (EPS)
Dilute_EPS2 = 1200000/(500000 + 25000)
print("Diluted Earnings per share (EPS) = ",'{:,.2f}'.format(Dilute_EPS2),"euros")

Number of common shares created if the options are exercised =  100,000 shares

Cash inflow if the options are exercised =  100,000.00 euros

Number of shares that can be purchased with these funds =  75,000 shares

Net increase in shares outstanding from exercising stock options =  25,000 shares

Diluted Earnings per share (EPS) =  2.29 euros


**Answer:**

    Number of common shares created if the options are exercised = 100,000

    Cash inflow if the options are exercised = €15 per share × 100,000 shares = €1,500,000

    Number of shares that can be purchased with these funds = €1,500,000 / €20 = 75,000 shares

    Net increase in common shares outstanding from the exercise of the stock options = 100,000 – 75,000 = 25,000 shares


<p> <br> </p>

$\qquad\text{diluted EPS} = \Large\frac{€1,200,000}{500,000 + 25,000} \small = €2.29$

<p> <br> </p>

A quick way to calculate the net increase in common shares from the potential exercise of stock options or warrants when the exercise price is less than the average market price is:

<p> <br> </p>

$\qquad\Large\Biggr[\frac{\text{AMP - EP}}{\text{AMP}}\Biggr] \small \times N $

<p> <br> </p>

where:
\
\
$\text{AMP}$ = average market price over the year
\
\
$\text{EP}$ = exercise price of the options or warrants
\
\
$\text{N}$ = number of common shares that the options and warrants can be converted into

<p> <br> </p>

for XXX:

$\qquad\Large\frac{€20 - €15}{€20} \small \times 100,000 \text{ shares} = 25,000 \text{ shares}$

In [74]:
# calculate the net increase in common shares from the potential exercise of stock options 
# or warrants when the exercise price is less than the average market price 
XXX = ((20-15)/20) *100000
print("Number of common shares created if the options are exercised = ",'{:,.0f}'.format(XXX),"shares")
print("")

Number of common shares created if the options are exercised =  25,000 shares



<hr>

# Module 30.4 Questions

<hr>

<hr>

**Question 1:** \
Hall Corp. had 100,000 shares of common stock outstanding at the beginning of the year. Hall issued 30,000 shares of common stock on May 1. On July 1, the company issued a 10% stock dividend. On September 1, Hall issued 1,000, 10% bonds, each convertible into 21 shares of common stock. What is the weighted average number of shares to be used in computing diluted EPS, assuming the convertible bonds are dilutive?


**Answer:** \
139,000


**Explanation:** \
The new stock is weighted by `8 / 12`. The bonds are weighted by `4 / 12` and are not affected by the stock dividend.

    basic shares = {[100,000 × (12 / 12)] + [30,000 × (8 / 12)]} × 1.10 = 132,000

    diluted shares = 132,000 + [21,000 × (4 / 12)] = 139,000
\
(Module 30.4, LOS 30.d)

<hr>

<hr>

**Question 2:** \
An analyst has gathered the following information about a company:

* 300,000 shares outstanding
* 100,000 warrants exercisable at €50 per share
* Average share price: €55
* Year-end share price: €60

How many shares should be used in computing diluted EPS?



**Answer:** \
309,091.


**Explanation:** \
Because the exercise price of the warrants is less than the average share price, the warrants are dilutive. Use the Treasury stock method to determine the denominator impact:

$\quad\Large\frac{55-50}{55} \small \times 100,000 \text{ shares} = 9,091 \text{ shares}$

\
Thus, the denominator will increase by 9,091 shares to 309,091 shares. The question asks for the total, not just the impact of the warrants. 
\
(Module 30.4, LOS 30.d)

<hr>

<hr>

**Question 3:** \
An analyst has gathered the following information about a company:

* 100,000 common shares outstanding from the beginning of the year
* Earnings: €125,000
* 1,000, 7\%, €1,000 par bonds convertible into 25 shares each, outstanding as of the beginning of the year
* Tax rate: 40%

The company's diluted EPS is closest to:


**Answer:** \
€1.25.


**Explanation:** \
First, calculate basic EPS:

€125,000100,000 = €1.25

Next, check if the convertible bonds are dilutive:

    numerator impact = (1,000 × 1,000 × 0.07) × (1 – 0.4) = €42,000

    denominator impact = (1,000 × 25) = 25,000 shares

    per share impact= €42,000 / 25,000 shares= €1.68

Because €1.68 is greater than the basic EPS of €1.25, the bonds are antidilutive. Thus, diluted EPS = basic EPS = €1.25.
\
(Module 30.4, LOS 30.d)

<hr>

<hr>

**Question 4:** \
An analyst has gathered the following information about a company:

* 50,000 common shares outstanding from the beginning of the year
* Warrants outstanding all year on 50,000 shares, exercisable at €20 per share
* Stock is selling at year-end for €25
* Average price of the company's stock for the year was €15

How many shares should be used in calculating the company's diluted EPS?


**Answer:** \
50,000.

**Explanation:** \
The warrants, in this case, are antidilutive. The average price per share of €15 is less than the exercise price of €20. The year-end price per share is not relevant. The denominator consists of only the common stock for basic EPS. 
\
(Module 30.4, LOS 30.d)
<hr>

# footnotes

<img src = "https://github.com/PachaTech/CFA-Level-1/blob/main/Financial%20Statement%20Analysis/Module%2030/pics/30_4_1.png?raw=true">

<img src = "https://github.com/PachaTech/CFA-Level-1/blob/main/Financial%20Statement%20Analysis/Module%2030/pics/30_4_2.png?raw=true">

<img src = "https://github.com/PachaTech/CFA-Level-1/blob/main/Financial%20Statement%20Analysis/Module%2030/pics/30_4_3.png?raw=true">

<img src = "https://github.com/PachaTech/CFA-Level-1/blob/main/Financial%20Statement%20Analysis/Module%2030/pics/30_4_4.png?raw=true">

<img src = "https://github.com/PachaTech/CFA-Level-1/blob/main/Financial%20Statement%20Analysis/Module%2030/pics/30_4_5.png?raw=true">

<img src = "https://github.com/PachaTech/CFA-Level-1/blob/main/Financial%20Statement%20Analysis/Module%2030/pics/30_4_6.png?raw=true">

<img src = "https://github.com/PachaTech/CFA-Level-1/blob/main/Financial%20Statement%20Analysis/Module%2030/pics/30_4_7.png?raw=true">

<img src = "https://github.com/PachaTech/CFA-Level-1/blob/main/Financial%20Statement%20Analysis/Module%2030/pics/30_4_8.png?raw=true">